core/templates/dev/head/pages/exploration_editor/editor_tab/state_content_editor_directive.html

```diff
@@ -25,14 +25,7 @@
     <div class="oppia-editable-section-mask protractor-test-state-edit-content">
     </div>
   </div>
-
-  <div ng-if="stateContentService.savedMemento && !stateContentService.savedMemento.isEmpty()">
-    <audio-translations-editor component-name="<[COMPONENT_NAME_CONTENT]>"
-                               subtitled-html="stateContentService.displayed"
-                               on-start-edit="onAudioTranslationsStartEditAction"
-                               on-change="onAudioTranslationsEdited">
-    </audio-translations-editor>
-  </div>
+ <subtitled-html-editor></subtitled-html-editor>
```

Comment:
Shouldn't there be something being passed into the directive?

Reply:
OK i'll update and test it as well